metadata.json

```diff
@@ -73,6 +73,12 @@
         "8"
       ]
     },
+    {
+      "operatingsystem": "Rocky",
+      "operatingsystemrelease": [
+        "8"
+      ]
+    },
```

Comment:
Can you please move this block bellow RedHat so that entries are sorted alphabetically? Thanks!

Reply:
Fixed